src/components/chip-set/chip-set.tsx

```diff
@@ -370,7 +394,7 @@ export class ChipSet {
                 {this.value.map(this.renderInputChip)}
                 <input
                     tabIndex={INPUT_FIELD_TABINDEX}
-                    type="text"
+                    type={this.inputType}
```

Comment:
![Image from iOS](https://user-images.githubusercontent.com/35954987/164023724-89994849-1e3b-43e8-a260-5ab53a7041f1.png)

## ⚠️ head up

We shouldn't merge this yet, until we know how to get rid of the extra 🔍 icon that iOS adds.

Reply:
https://stackoverflow.com/questions/23296072/html5-search-input-magnifying-glass-override/23296152

Reply:
The last changes fixed an extra X button and the background colour 
 ![Image from iOS (1)](https://user-images.githubusercontent.com/50618208/164193163-836e67a3-0706-4b3c-83be-2a06553736b2.png)
![Image from iOS](https://user-images.githubusercontent.com/50618208/164193259-6bee5281-a263-4e2e-a7bd-c791b5b98e42.png)

Reply:
So, what's the status here @Kiarokh? OK to merge after review, or not?

Reply:
I think it's alright.